git_ignore.qmd

```diff
@@ -0,0 +1,142 @@
+---
+title: "The `.gitignore`"
+---
+
+## Overview
+
+The purpose of the `.gitignore` is evident in its name: anything included in it is *ignored by `git`*. This can be useful if you want to include data in your local version of your repository but don't want to risk sharing that data by committing it to a GitHub repository you plan to make public at some point. This file exists at the top level of every repository and can be customized however is most useful to you and your collaborators.
+
+## Creating a `.gitignore`
+
+When you first create a repository you will have the opportunity to select a template `.gitignore` based on the coding language you plan on using. In the "New Repository" process, scroll down to just below where you can choose to add a README.
+
+<p align="center">
+<img src="images/git_ignore/ignore-2-new-repo-p2.png" width = "70%"/>
+</p>
+
+Underneath the title "Add .gitignore" there is a dropdown menu for various templates that starts at "None". If you click this dropdown menu you can type in keywords that will help you identify the template that you want to use.
+
+<p align="center">
+<img src="images/git_ignore/ignore-3-template-search.png" width = "70%"/>
+</p>
+
+Unfortunately for R users, typing just "R" returns all templates that contain the letter 'r' so you will need to scroll a bit to find the option that is actually for the R Statistical Environment. Once you find the option you want, click it and the template dropdown should change from ".gitignore template: None" to ".gitignore template: \<your pick\>".
+
+<p align="center">
+<img src="images/git_ignore/ignore-4-template-pick.png" width = "70%"/>
+</p>
+
+If you create a repository without choosing a `.gitignore` template, don't worry! A less detailed one will be created automatically when you clone your repository to your local computer. You can then add whatever information you want and make it contain the same information as the starter template.
```

Comment:
RStudio does create / modify a .gitignore file when you create an RStudio project, but it is not true when the command line or other tools.

Reply:
Good caveat, I'll make this tweak once you're finished with edits